scripts/src/main/resources/scripts/command/helm

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+# Call helm with specified arguments.
+function doRun() {
+  doSetup silent
+  doRunCommand "${HELM_HOME}/helm ${@}"
+}
+
+function doSetup() {
+  # Check if HELM_HOME is already set
+  if [ -z "${HELM_HOME}" ]
+  then
+    HELM_HOME="${DEVON_IDE_HOME}/software/helm"
+  fi
+
+  if [ ! -d ${HELM_HOME} ] || [ "${1}" != "silent" ]
+  then
+    # Get leatest release
+    if [ -z "${HELM_VERSION}" ]
+    then
+      doEcho "Getting latest release..."
+      HELM_VERSION=$(curl -LsH "Accept: application/vnd.github.v3+json" https://api.github.com/repos/helm/helm/releases/latest | awk -F':' '/tag_name/ {print $2}' | awk -F'"' '{print $2}')
+    fi
+
+    doInstall "-" "${HELM_HOME}" "helm" "${HELM_VERSION}"
+  fi
+
+  if command -v helm &> /dev/null 
```

Comment:
just for curiosity - what happened without this if?
Actually at this point helm must be present or the above code did not do its job.
Du you fear any PATH variable issues?
In other commandlets we do not have such check...

Reply:
I removed the wrong if case. Same in Terraform